libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFManagedPreferences.m

```diff
@@ -78,6 +82,9 @@ - (id)init {
                                                           weakSelf.rawPreferences = [[NSUserDefaults standardUserDefaults] dictionaryForKey:kManagedConfigurationKey];
```

Comment:
Can you make the change below here, as well?  This notification handler refreshes the MDM preferences when user defaults values change.